services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouterToPMServiceImpl.java

```diff
@@ -81,4 +81,10 @@ public void updatePort(String projectid, String portId, PortEntity portEntity) {
         HttpEntity<PortWebJson> request = new HttpEntity<>(new PortWebJson(portEntity));
         restTemplate.put(portManagerServiceUrl, request, PortWebJson.class);
     }
+
+    @Override
+    public void updateL3Neighbors(String projectid, List<String> gatewayPorts) {
+        String portManagerServiceUrl = portUrl + "/project/" + projectid + "/update-l3-neighbors";
+        //restTemplate.put(portManagerServiceUrl, request, PortWebJson.class);
+    }
```

Comment:
@kevin-zhonghao please prepare a `RouterUpdateInfo` object here for request body. @chenpiaoping's code for this API requires `RouterUpdateInfo` object.

Reply:
Added it